pynuodb/session.py

```diff
@@ -67,7 +66,7 @@ def __str__(self):
 class Session:
 
     __AUTH_REQ = "<Authorize TargetService=\"%s\" Type=\"SRP\"/>"
-    __SRP_REQ = "<SRPRequest ClientKey=\"%s\" Cipher=\"RC4\" Username=\"%s\"/>"
+    __SRP_REQ = "<SRPRequest ClientKey=\"%s\" Cipher=\"RC4\" Username=\"%s\"/>" #Why is this hard coded...
```

Comment:
It was meant to be a note to myself but, didn't get around to resolving before pushing. I think this should be dynamically built from the options passed into the connection